### PR TITLE
enable crash reporting on nightly

### PR DESCRIPTION
**Pull request by wolverineks**
https://emurgo.atlassian.net/browse/YOMO-401

### CODE REVIEW

File: src/hooks/index.ts [hunk 1]
```diff
@@ -52,19 +52,13 @@ import {
...
   const data = await storage.getItem(crashReportsStorageKey)
-  if (data != null) {
-    const parsed = JSON.parse(data)
-    if (typeof parsed === 'boolean') {
-      return parsed
-    }
-  }
-
-  return false
+  return parseBoolean(data) ?? false
```
Comment (wolverineks) [inline src/hooks/index.ts:61]: cleanups

File: src/legacy/actions.ts [hunk 2]
```diff
@@ -109,14 +109,10 @@ export const initApp = () => async (dispatch: Dispatch<any>, getState: any) => {
...
 
-  if (isNightly()) {
-    dispatch(setAppSettingField(APP_SETTINGS_KEYS.SEND_CRASH_REPORTS, true))
-  }
-
   await dispatch(reloadAppSettings())
   await dispatch(initInstallationId())
 
-  const crashReportsEnabled = await getCrashReportsEnabled()
+  const crashReportsEnabled = isNightly() ? true : await getCrashReportsEnabled()
```
Comment (wolverineks) [inline src/legacy/actions.ts:115]: this is the fix

File: src/legacy/appSettings.ts [hunk 3]
```diff
@@ -12,7 +12,6 @@ export const APP_SETTINGS_KEYS = {
   CUSTOM_PIN_HASH: 'customPinHash',
   ACCEPTED_TOS: 'acceptedTos',
   LANG: 'languageCode',
-  SEND_CRASH_REPORTS: 'sendCrashReports',
```
Comment (wolverineks) [inline src/legacy/appSettings.ts:15]: cleanup

File: src/legacy/state.ts [hunk 4]
```diff
@@ -18,7 +18,6 @@ export type State = {
     acceptedTos: boolean
     installationId: string | null | undefined
     customPinHash: string | null | undefined
-    sendCrashReports: boolean
```
Comment (wolverineks) [inline src/legacy/state.ts:21]: cleanup

File: src/yoroi-wallets/utils/parsing.ts [hunk 5]
```diff
@@ -64,3 +64,18 @@ export const parseAmountDecimal = (amount: string, token: Token): BigNumber => {
 
   return value
 }
+
```
Comment (wolverineks) [inline src/yoroi-wallets/utils/parsing.ts:67]: cleanup

**Review by banklesss — APPROVED** [review]

File: src/legacy/state.ts [hunk 6]
```diff
@@ -27,7 +26,6 @@ export const getInitialState = (): State => ({
     acceptedTos: false,
```
Comment (stackchain) [inline src/legacy/state.ts:26]: just realized `acceptTos` needs clean up too

**Review by stackchain — APPROVED** [review]

**Pull request merged**